### ISSUE TITLE
Basic integrations with Zapier & Integromat

### ISSUE THREAD

**Issue by dsecareanu** [mettle/sendportal#64]
It would be great if the platform would feature integrations with Zapier & Integromat (for non technical users) as an easier alternative to API driven integrations (would make adoption easier I believe).

**Comment by JonoB** [comment]
Yes!  Our API is quite comprehensive at the moment, so we'll definitely look at this in the future. Not quite ready yet though.
